script.js

```diff
@@ -0,0 +1,959 @@
+/* eslint-disable no-continue */
+// The above es lint rule is disabled, because if continue is not used
```

Comment:
Thanks for the explanation!! For future reference, typically explanations go above the code they are explaining.

Reply:
Updated